### PR TITLE
CB-11

### PR DESCRIPTION
**Pull request by efesabanogluu**

### CODE REVIEW

**Review by kocasariumut — APPROVED** [review]
works as intended

**Review by hamzaakyildiz — APPROVED** [review]

**Review by onurcanavci — APPROVED** [review]

**Review by ramazanbulut — APPROVED** [review]

**Review by asi345 — APPROVED** [review]
I think it is fine.

**Review by mrabiabrn — APPROVED** [review]

**Review by erencanuysal — APPROVED** [review]

File: practice-app/location/location.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+from flask import Blueprint, request, jsonify, abort
+from ..database import mongo
+import requests
+import datetime
+
+relatedLocations = []
+location_bp = Blueprint('Create Location', __name__)
+@location_bp.route('/api/locations/<string:address>', methods=['POST'])
+def createLocation(post,username):
```
Comment (CodeBlo) [inline practice-app/location/location.py:9]:
```suggestion
def createLocation(address):

```

Comment (CodeBlo) [inline practice-app/location/location.py:9]:
```suggestion
def createLocation():

```

File: practice-app/location/location.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+from flask import Blueprint, request, jsonify, abort
+from ..database import mongo
+import requests
+import datetime
+
+relatedLocations = []
+location_bp = Blueprint('Create Location', __name__)
+@location_bp.route('/api/locations/<string:address>', methods=['POST'])
```
Comment (CodeBlo) [inline practice-app/location/location.py:8]:
```suggestion
@location_bp.route('/api/locations', methods=['POST'])

```

**Review by CodeBlo — APPROVED** [review]

**Review by onurcanavci — APPROVED** [review]

**Pull request merged**